sdk/src/lib.rs

```diff
@@ -30,8 +30,6 @@ pub mod utils;
 pub use crypto;
 pub use packable;
 pub use primitive_types::U256;
-#[cfg(feature = "url")]
-pub use url::Url;
```

Comment:
Why can it not be that way? And why specifically Url?
I think it was done this way because it's part of our public API, people should use it from us, otherwise if we update it on our side it would be breaking change for them. Or something like this.

Reply:
If there is a breaking change in Url while we are re-exporting it, we must release a breaking update as well. De-coupling doesn't change this since we expose the types anyway in our public API.

Reply:
What motivated the change in the context of this PR?